src/AMP.php

```diff
@@ -49,7 +49,9 @@ class AMP
     // The StandardFixPass should be after StandardScanPass
     public $passes = [
         'Lullabot\AMP\Pass\PreliminaryPass', // Removes user blacklisted tags
-        'Lullabot\AMP\Pass\SugarImgTagTransformPass',
+        'Lullabot\AMP\Pass\ImgTagTransformPass',
```

Comment:
We are actually not planning to maintain a fork. @FuzzyTree  is working on submiting PRs back to the main repo to get his changes accepted.